README.md

```diff
@@ -62,6 +62,28 @@ defmodule UserSchema do
 end
 ```
 
+Another possibility is avoiding creating files for schema at all. This can be done by importing `Parameter.Schema` and use the `param/2` macro. This is useful for adding params on Phoenix Controllers for example:
```

Comment:
```suggestion
Another possibility is avoiding creating files for a schema at all. This can be done by importing `Parameter.Schema` and using the `param/2` macro. This is useful for adding params in Phoenix controllers. For example:
```